services/scabbard/libscabbard/src/client/mod.rs

```diff
@@ -29,18 +29,69 @@ use super::protocol::SCABBARD_PROTOCOL_VERSION;
 
 pub use error::ScabbardClientError;
 
+/// Builder for building a ScabbardClient.
+#[derive(Default)]
+pub struct ScabbardClientBuilder {
```

Comment:
Generally we define builders in a `builder` submodule and `pub use` them at the level of the thing they construct. This keeps this higher level module less cluttered.

Reply:
Made a builder submodule